x-pack/plugins/infra/server/plugin.ts

```diff
@@ -171,6 +187,10 @@ export class InfraServerPlugin implements Plugin<InfraPluginSetup> {
     ]);
 
     initInfraServer(this.libs);
+    initLogViewRoutes({
+      framework,
+      getLogViewsService: async () => (await core.getStartServices())[2].logViews,
```

Comment:
Just to make sure my own understanding is correct:

We pass through a `getLogViewsService` function here, whereas with the other routes we would call `getStartServices` on `libs` to access `logViews`, this is just to make the focus / functionality clearer and more specific?

Reply:
Hm, this was probably just relatively early code in which I had more inhibition to specify the full start accessor as a dependency. That it stood out to you indicates that I should probably change it to align with the rest?

Reply:
It’s a small nit really. Once I checked the function it made sense, but aligning them is probably slightly less jarring when looking at everything for the first time.